### DISCUSSION TITLE
How to create a new window from command line?

### DISCUSSION THREAD

**Discussion by leira** [wezterm/wezterm#1477] [Q&A]
I added `unix` domain, and `connect_automatically`. Now every time I run `wezterm`, it connects to the `unix` domain, duplicate every window in the domain. What is the command to add a new `wezterm` window?

```lua
  unix_domains = {
    {
      name = "unix",
      connect_automatically = true,
    },
  },
```

**Comment by wez** [comment]
FWIW, I'm recommending the use of a new option instead of `connect_automatically`; take a look at this comment:
https://github.com/wez/wezterm/issues/1299#issuecomment-993770970
(I'll update the docs once I cut a new release, as it will make it much easier to write the docs that way)

Regarding your question: take a look at `wezterm cli --help`.  You can use `wezterm cli spawn` to create new windows/tabs, or `wezterm cli split-pane` to make new panes.

**Comment by leira** [comment]
Thanks for the answer! I will take a look at `connect_automatically` when it is in nixos-unstable.

The workflow I'm thinking about is to use `rofi` to create a new window of wezterm into a unix domain, rather than start a new domain. `wezterm cli spawn` needs to be run within a wezterm instance, if I understand it correct. How to `cli` into a specific domain?

**Reply by wez** [reply]
If you run `wezterm cli` from outside of wezterm, it will look up the unix socket path from the config file and connect to it.  You should be able to run `wezterm cli spawn --new-window` to get the behavior that you want.
